### PR TITLE
GRVDEV-147 Create Terraform ECS module

### PR DESCRIPTION
**Pull request by jeremyodle**
Pull request created in: https://gravicore.atlassian.net/browse/GRVDEV-147 by the Git Integration for Jira app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ This is a collection of reusable Terraform modules for Gravicore's cloud automat
...
 | [Data Transfer](aws/data-transfer)                   | A module for deploying a data transfer solution utilizing [AWS DataSync](https://aws.amazon.com/datasync/) and [AWS Snowball](https://aws.amazon.com/snowball/).                                                                                                                                                                           |
 | [Datadog Agent](aws/datadog-agent)                   | A module for deploying a centralized logging solution leveraging [Datadog](https://www.datadoghq.com/).                                                                                                                                                                                                                                    |
+| [ECS](aws/ecs)                                       | A module for deploying fargate containers using [AWS Elastic Container Service (ECS)](https://aws.amazon.com/ecs/) service.                                                                                                                                                                                                                |
```
Comment (darlanmoraes) [inline README.md:28]: Add datadog and camunda to the list

**Review by TysonBeck — APPROVED** [review]

**Review by TysonBeck — CHANGES_REQUESTED** [review]
Need to merge master int the branch as is currently 4 commits behind

**Comment by jeremyodle** [comment]
> Need to merge master int the branch as is currently 4 commits behind

@TysonBeck @darlanmoraes I would suggest a rebase on this given this is a feature branch to keep the history cleaner.

File: aws/camunda/java/api/src/main/java/io/gravicore/loc/Application.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package io.gravicore.loc;
```
Comment (jeremyodle) [inline aws/camunda/java/api/src/main/java/io/gravicore/loc/Application.java]: @darlanmoraes Should there be a reference to LOC?

Comment (jeremyodle) [reply aws/camunda/java/api/src/main/java/io/gravicore/loc/Application.java]: @darlanmoraes Perhaps change `.loc` to `.camunda`?

File: aws/camunda/module/module.tf [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+  }
+}
+
+# ----------------------------------------------------------------------------------------------------------------------
+# Module Standard Variables
+# ----------------------------------------------------------------------------------------------------------------------
+
+variable "name" {
+  type        = string
+  default     = "ecs"
```
Comment (jeremyodle) [inline aws/camunda/module/module.tf]: The module name should be `camunda`?

File: aws/camunda/java/api/src/main/java/io/gravicore/loc/Application.java (hunk 2 above)
Comment (darlanmoraes) [reply aws/camunda/java/api/src/main/java/io/gravicore/loc/Application.java]: @jeremyodle I've changed to `app` and `api` to differentiate each type of application. What do you think? If you think that there's the need for `camunda`, maybe we can use `.camunda.app` and `.camunda.api`

**Review by TysonBeck — APPROVED** [review]

**Pull request merged**